wp-content/themes/chinapower/template-parts/content-podcasts.php

```diff
@@ -26,34 +28,44 @@
 ?>
 
 <article id="post-<?php the_ID(); ?>" <?php post_class($classes); ?>>
-	<?php if ( $sticky ) { ?>
-	<div class="entry-thumbnail col-xs-12 col-md-4">
-		<iframe width="246" height="245" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/<?php echo $soundcloudID; ?>&amp;auto_play=false&amp;hide_related=false&amp;show_comments=true&amp;show_user=true&amp;show_reposts=false&amp;visual=true"></iframe>
-	</div>
-	<div class="sticky-container col-xs-12 col-md-8">
+	<?php if ( $sticky and ($soundcloudID or $megaphoneEmbedURL )) { ?>
```

Comment:
```suggestion
	<?php if ( $sticky && ($soundcloudID || $megaphoneEmbedURL )) { ?>
```

`and` and `or` are not valid operators in PHP, so I'm swapping them to `&&` and `||` respectively.

Reply:
@urchykli I'm going to resolve this since my comment solves this problem, but flagging it as an FYI for you.